datacite/schema/example.json

```diff
@@ -0,0 +1,117 @@
+{
+    "identifier": {
+        "identifier": "10.5072/example-full",
+        "type": "DOI"
+    },
+    "creators": [
+        {
+            "name": "Miller, Elizabeth",
+            "identifier": {
+                "identifier": "0000-0001-5000-0007",
+                "scheme": {
+                    "name": "ORCID",
+                    "uri": "http://orcid.org/"
+                }
+            }
+        }
+    ],
+    "titles": [
+        {
+            "title": "Full DataCite XML Example",
+            "language": "en-us"
+        },
+        {
+            "title": "Demonstration of DataCite Properties.",
+            "type": "Subtitle",
+            "language": "en-us" 
+        }
+    ],
+    "descriptions": [
+        {
+            "type": "Abstract",
+            "language": "en-us",
+            "description": "XML example of all DataCite Metadata Schema v3.1 properties."
+        }
+    ],
+    "publisher": "DataCite",
+    "publicationYear": 2014,
+    "subjects": [
+        {
+            "subjects": "000 computer science",
+            "scheme": {
+                "name": "dewey",
+                "uri": "http://dewey.info/"
+            },
+            "language": "en-us"
+        }
+    ],
+    "contributors": [
+        {
+            "name": "Starr, Joan",
+            "type": "ProjectLeader",
+            "identifier": {
+                "identifier": "0000-0002-7285-027X",
+                "scheme": {
+                    "name": "ORCID",
+                    "uri": "http://orcid.org/"
+                }
+            },
+            "affliation": "California Digital Library"
+        }
+    ],
+    "language": "en-us",
+    "alternateIdentifiers": [
+        {
+            "identifier": "http://schema.datacite.org/schema/meta/kernel-3.1/example/datacite-example-full-v3.1.xml",
+            "type": "URL"
+        }
+    ],
+    "relatedIdentifiers": [
+        {
+            "type": "URL",
+            "relation": "HasMetadata",
+            "scheme": {
+                "name": "citeproc+json",
+                "uri": "https://github.com/citation-style-language/schema/raw/master/csl-data.json"
+            }
+        }
+    ],
+    "dates": [
+        {
+            "date": "2014-10-17",
+            "type": "Updated"
+        },
+        {
+            "date": "1997-07-16T19:20+15:00",
+            "type": "Updated"
+        }
+    ],
+    "rights": [
+        {
+            "name": "CC0 1.0 Universal",
+            "uri": "http://creativecommons.org/publicdomain/zero/1.0/"
+        }
+    ],
+    "resourceType": {
+        "general": "Software",
+        "type": "XML"
+    },
+    "sizes": [ "3KB" ],
+    "formats": [ "application/xml" ],
+    "version": "3.1",
```

Comment:
Do we need to add the version inside the JSON? Just thinking it's not part of the standard, and a document can be compliant with multiple schema versions, and it's kind of already defined by the schema URL.